Recruit-Training/Advanced-Recruit-Training/src/BatteryStateDisplayService.cpp

```diff
@@ -25,9 +26,29 @@ void BatteryStateDisplayService::handleBatteryDataReceived(const BatteryData& ba
 {
     batteryStateOfChargeService_.addData(batteryData);
 
+    double milisec = batteryStateOfChargeService_.timeWhenChargedOrDepleted().msec();
+    QString addZero;
+    if(milisec < 100){
+        addZero = "0";
+    }
+
+    if(milisec < 10){
+        addZero = "00";
+    }
+
     QTextStream(stdout) << "Voltage: " << batteryData.voltage
         << " Current: " << batteryData.current
-        << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed() << endl;
+        << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed();
+    if(batteryStateOfChargeService_.isCharging()){
+        QTextStream(stdout) << " Time until charged: " << batteryStateOfChargeService_.timeWhenChargedOrDepleted().toString() <<"." << addZero << milisec <<endl;
```

Comment:
When lines are too long, just place some of the line below it with the ENTER key.